addons/mail/static/src/js/activity.js

```diff
@@ -682,6 +686,14 @@ var KanbanActivity = BasicActivity.extend({
      */
     _render: function () {
         var $span = this.$('.o_activity_btn > span');
+        // when no goupby on kanban, it will not call renderElement so update activity icon manually.
+        if (this.recordData.decoration_type) {
+            $span.removeClass('fa-clock-o');
```

Comment:
I would like to see tests (or test modifications) that express the new behaviors (see chatter_tests.js).

Reply:
@mge-odoo Done.